README.md

```diff
@@ -1,13 +1,26 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
```

Comment:
Overall, great work Petra 😄 your code is clean and easy to follow! 
Would love some more comments on your code, and please look through your endpoints again as some of them do not seem to work on my end. Otherwise, great 🙌